File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] prod = new int[nums.length];

        for(int i = 0; i < nums.length; i++) {
            int prodcs = 1;

            for(int j = 0; j < nums.length; j++){
              if(i!=j)  prodcs = prodcs * nums[j];
            }

                 prod[i] = prodcs;


        }

        return prod;
        
    }
}  

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        let longest = 0;

        for (let i = 0; i < nums.length; i++) {
            let current = nums[i];
            let count = 1;
            let found = true;

            while (found) {
                found = false;

                for (let j = 0; j < nums.length; j++) {
                    if (nums[j] === current + 1) {
                        current = nums[j];
                        count++;
                        found = true;
                        break;
                    }
                }
            }

            longest = Math.max(longest, count);
        }

        return longest;
    }
}